types/src/main.ts

```diff
@@ -2611,6 +2616,15 @@ export interface TextDocumentContentChangeEvent {
 	text: string;
```

Comment:
Should we update the comments for `text`? In FullSync mode, `text` is always the content of the full document from client side but now in incremntal mode, it's the change for the corresponding range. Another thing we need to make sure that Client sends only content change but not full text in incremtnal mode.

Reply:
https://github.com/Microsoft/vscode-languageserver-node/blob/053149a76e7ec147347e41a5847a1cc4ccb3d541/client/src/client.ts#L936-L940

It's already guaranteed that when it's incremental, it sends the change event but not the full doc. So we only need to update the documentation.

Reply:
Will update the documentation in the next commit to the PR :)